Resources/Prototypes/DeltaV/Recipes/Lathes/medical.yml

```diff
@@ -22,3 +22,13 @@
   materials:
     Plastic: 150
     Steel: 50
+
+- type: latheRecipe
+  id: WeaponHealGun
+  result: WeaponHealGun
+  completetime: 1.5
+  materials:
```

Comment:
parent it to BaseWeaponRecipe so it cant be made instantly (remove completeTime: 1.5) and is in weapons category